subsys/random/rand32_ctr_drbg.c

```diff
@@ -51,16 +51,11 @@ static int ctr_drbg_initialize(void)
 {
 	int ret;
 
-	/* Only one entropy device exists, so this is safe even
-	 * if the whole operation isn't atomic.
-	 */
-	entropy_driver = device_get_binding(DT_CHOSEN_ZEPHYR_ENTROPY_LABEL);
-	if (!entropy_driver) {
-		__ASSERT((entropy_driver != NULL),
-			"Device driver for %s (DT_CHOSEN_ZEPHYR_ENTROPY_LABEL) not found. "
-			"Check your build configuration!",
-			DT_CHOSEN_ZEPHYR_ENTROPY_LABEL);
-		return -EINVAL;
+	if (!device_is_ready(entropy_driver)) {
+		__ASSERT(0, "Device driver for %s not ready. "
```

Comment:
It would be nice to see this changed to "Entropy device %s not ready". We have a legacy in zephyr of mixing up drivers with devices, and calling `struct device` instances "drivers", which I think is a misnomer that would be nice to improve wherever we can while we're "modernizing".

Reply:
adjusted, pls check